pkg/chartmuseum/server/multitenant/handlers.go

```diff
@@ -130,6 +130,17 @@ func (server *MultiTenantServer) getIndexFileRequestHandler(c *gin.Context) {
 	c.Data(200, indexFileContentType, indexFile.Raw)
 }
 
+func (server *MultiTenantServer) headIndexFileRequestHandler(c *gin.Context) {
+	repo := c.Param("repo")
+	log := server.Logger.ContextLoggingFn(c)
+	indexFile, err := server.getIndexFile(log, repo)
```

Comment:
indexFile defined but not used here ?

Reply:
Removed

Reply:
Always 200 even if  index file not found ? Does it need to  `stat` first ?

Reply:
@scbizu sorry can you describe me your full proposa for this method to avoid over commit?

Thanks and sorry to bother you

Reply:
In my opinion the first commit was enough to cover error on file index generation

Dunno if stat can be performed as it basically it's a generated string

Reply:
Your welcome , just take it easy . It seems me OK for me to return 200 here .